catroid/src/androidTest/java/org/catrobat/catroid/test/devices/arduino/ArduinoSettingsTest.java

```diff
@@ -79,7 +81,8 @@ public void testIfArduinoBricksAreEnabledIfItItUsedInAProgram() throws IOExcepti
 
 		assertFalse(SettingsFragment.isArduinoSharedPreferenceEnabled(context));
 
-		ProjectManager.getInstance().loadProject(project.getDirectory(), context);
+		final Lazy<ProjectManager> projectManager = inject(ProjectManager.class);
+		projectManager.getValue().loadProject(project.getDirectory());
```

Comment:
Consistency:
Sometimes  we have the projectManager as variable for the whole file and sometimes locally in every test.
Would it not make sense to use the same one for the all the test or could it lead to errors?

Reply:
@juliajulie95 Effectively it would be the same. In some cases, e.g. classes/files that have to be serialized, the reference always have to be a local variable in the method otherwise the class cannot be serialized and leads to errors. 

The decision whether to include it as a local variable or a member variable is also made based on the frequency of the access, in order to reduce the amount of duplicate code.